cwms-data-api/src/main/java/cwms/cda/data/dto/Project.java

```diff
@@ -0,0 +1,317 @@
+package cwms.cda.data.dto;
+
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.JsonV2;
+import java.time.Instant;
+
+@JsonDeserialize(builder = Project.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+@FormattableWith(contentType = Formats.JSON, formatter = JsonV2.class)
+public class Project extends CwmsDTO {
+
+    private final String name;
+    private final Double federalCost;
+    private final Double nonFederalCost;
```

Comment:
What do you think about the cost classes?   Is Double sufficient?  Or should the DTO hold them as BigDecimal?

Reply:
I think double should be fine. Might be worth a review of existing data to see what the actual numbers are but the database type is binary_double so BigDecimal isn't going to get us anything here.